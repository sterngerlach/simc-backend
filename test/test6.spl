
{
    int i = 5;

    sprint("1: \n");

    do {
        sprint("\ti: ");
        iprint(i);
        sprint("\n");
        i -= 1;
    } while (i < 0);
    
    sprint("2: \n");

    do {
        sprint("\ti: ");
        iprint(i);
        sprint("\n");
        i -= 1;
    } while (i >= 3);

    sprint("end\n");
}

