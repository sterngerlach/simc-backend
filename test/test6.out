1: 
	i: 5
2: 
	i: 4
	i: 3
end
